packages/components/src/styles/colors.js

```diff
@@ -52,6 +52,8 @@ module.exports = {
     enabled: '#FFFFFF',
     focus: '#FFFFFF',
     disabled: '#D8D8D8',
+    inactive: 'rgba(0, 0, 0, 0.3)',
```

Comment:
Design said that `inactive` is the same as `enabled`. They have a missnaming on DSM. So we can remove this new inactive variable. 

We don't have to use fill-form colors for text color. This is a different property. Colors defined under fill-forms should be used only on the input box, not the content. We'll be using this fill-form property mainly on background colors.